app/constants/report-types.js

```diff
@@ -0,0 +1,8 @@
+module.exports.getReportTypes = () => {
+    return {
+        'Payment request statuses': 'payment-requests',
+        'Combined transaction report': 'transaction-summary',
+        'Suppressed payment requests': 'suppressed-payments',
+        'AP Listing Report': 'ap-listing'
+    }
+}
```

Comment:
Love how you've used this - but just for consistency I'd rewrite as 

const getReportTypes = () => {
  ...
}

module.exports = {
  getReportTypes
}

Just cause thats what we do elsewhere